packages/@aws-cdk/aws-stepfunctions-tasks/lib/sagemaker-create-endpoint-config-task.ts

```diff
@@ -0,0 +1,152 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Stack } from '@aws-cdk/core';
+import { getResourceArn } from './resource-arn-suffix';
+import { DataCaptureConfig, ProductionVariants } from './sagemaker-task-base-types';
+
+/**
+ *  @experimental
+ */
+export interface SagemakerCreateEndpointConfigTaskProps {
+
+    /**
+     * The request accepts the following data in JSON format.
+     *
+     * @default - None
+     */
+    readonly dataCaptureConfig?: DataCaptureConfig;
```

Comment:
I can't see this as a supported parameter here - https://docs.aws.amazon.com/step-functions/latest/dg/connect-sagemaker.html

Reply:
https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_CreateEndpointConfig.html

I see it here.

Reply:
Not all parameters of the API is supported by step functions. You can only configure the ones that are explicitly mentioned in the stepfunctions document (that I linked above).